src/api/donations/donations.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@nestjs/common';
+import { CreateDonationInput } from './dto/create-donation.input';
+import { PrismaService } from 'nestjs-prisma';
+
+type CreateDonationInputWithDonorId = CreateDonationInput & {
+  donorId: number;
+};
+
+@Injectable()
+export class DonationsService {
+  constructor(private prisma: PrismaService) {}
+  findAllByProjectId(projectId: number) {
+    return this.prisma.donation.findMany({ where: { projectId } });
+  }
```

Comment:
We may also need a find all by org id

Reply:
I think for now we can just have per project.